### PR TITLE
mock navigateTo

### PR DESCRIPTION
**Pull request by EmmaLRussell**
Initialise mock of navigateTo in vi.hoisted so it actually gets used...

### CODE REVIEW

File: tests/unit/components/ParameterForm.spec.ts [hunk 1]
```diff
@@ -120,7 +129,7 @@ describe("parameter form", () => {
       },
     });
 
-    const component = await mountSuspended(ParameterForm, {
+    const component = mount(ParameterForm, {
```
Comment (EmmaLRussell) [inline tests/unit/components/ParameterForm.spec.ts:132]: This change might not be necessary.

Comment (david-mears-2) [reply tests/unit/components/ParameterForm.spec.ts:132]: Yep, confirmed we can switch this back:

```suggestion
    const component = await mountSuspended(ParameterForm, {
```

File: tests/unit/components/ParameterForm.spec.ts [hunk 2]
```diff
@@ -53,8 +53,17 @@ const selectParameters = [
 ];
 
 const metadata = { modelVersion: "0.0.0", parameters: [...selectParameters, globeParameter] } as Metadata;
+// Need to do this in hoisted - see https://developer.mamezou-tech.com/en/blogs/2024/02/12/nuxt3-unit-testing-mock/
```
Comment (david-mears-2) [inline tests/unit/components/ParameterForm.spec.ts:56]: Thanks for linking this

**Review by david-mears-2 — APPROVED** [review]
Incredible, thanks so much for finding this

File: tests/unit/components/ParameterForm.spec.ts (hunk 1 above)
Comment (david-mears-2) [reply tests/unit/components/ParameterForm.spec.ts]: I committed this suggestion

**Comment by EmmaLRussell** [comment]
Closing as will be superseded by another branch.

**Pull request closed without merging**